log/keyvalue.go

```diff
@@ -32,70 +44,286 @@ const (
 )
 
 // A Value represents a structured log value.
-type Value struct{} // TODO (#4914): implement.
+type Value struct {
```

Comment:
I think that that it would be good to document that an empty value is valid.

Moreover, maybe we should add `Empty()` and `EmptyValue()` factory functions?
Personally, I do not find it necessary, but you can consider creating an issue if you think that it needs to be discussed..

Reply:
Better to create a separate issue and marked as blocked by https://github.com/open-telemetry/opentelemetry-specification/pull/3853

Reply:
I created https://github.com/open-telemetry/opentelemetry-go/issues/4953